clientlibs/js/src/UpgradeClient.ts

```diff
@@ -23,44 +23,51 @@ import * as uuid from 'uuid';
 export default class UpgradeClient {
   // Endpoints URLs
   private api = {
-    init: null,
-    getAllExperimentConditions: null,
-    markExperimentPoint: null,
-    setGroupMemberShip: null,
-    setWorkingGroup: null,
-    failedExperimentPoint: null,
-    getAllFeatureFlag: null,
-    log: null,
-    altUserIds: null,
-    addMetrics: null,
+    init: null as string,
```

Comment:
`null as string` is a bug waiting to happen, we can't be doing that, it makes me nervous that it's a band-aid for something somewhere else, but I think these can simply be empty strings instead of `null` right?